### ISSUE TITLE
Analyze the code

### ISSUE THREAD

**Issue by LaurentLienhard** [HowIAutomatedThis/HIATModuleTemplate#8]
Use [PSScriptAnalyzer](https://github.com/PowerShell/PSScriptAnalyzer) ?

**Comment by gaelcolas** [comment]
Of course! You can also look at https://github.com/dsccommunity/DscResource.AnalyzerRules and https://github.com/dsccommunity/DscResource.Test
